rfc/2022/2022-07-14_Platform_Security_001_Service_Account_Guidance.md

```diff
@@ -0,0 +1,112 @@
+
+*Authenticator Management*
+
+The Account Manager is responsible for the creating, maintaining, safeguarding, and rotating of the service account authenticator. 
+
+In the case of passwords, the Account Manager must ensure that the password for the service account meets minimum password requirements for length and complexity as dictated by VA policy.  Per IA-5(1), the VA requires a case sensitive 12-character password comprised of upper-case letters, lower case letters, numbers and special characters, including at least one of each category.
+
+Authenticators should never be stored in personal password managers.  It is recommended that authenticators be stored in Parameter Store.
+
+The Account Manager can grant (or revoke) access to the authenticator at their discretion amongst their team as necessary for those with a legitimate need to know.  Per IA-5, VA policy requires that service account authenticators be rotated at a minimum of every 3 years, or upon any member of the team or group with access to it leaving or no longer requiring access to the authenticator.  The Account Manager is responsible for providing access to the newly rotated authenticator as needed and ensuring that any application, service, or other process that rely on its use is updated.  
+
+*Intended Usage*
+
+The Account Manager is responsible for ensuring the service account is only used for the intended purpose as stated in the initial request for authorization.  Service accounts shall not be used interactively by an individual or used beyond their intended scope.  If a need arises to use a service account interactively for troubleshooting, approval from the ISO or designee (OCTO-DE VA employee) must be obtained and Platform Security should be CC’ed on the request.  If a need arises to either extend the intended usage of the service account beyond the initial scope, or reduce the scope of the intended usage, a new request documenting the change in intended usage must be submitted and approved by the ISO or designee (OCTO-DE VA employee).
+
+*Documentation and Tracking*
+
+The Account Manager is responsible for documenting, at a minimum, the following information for service accounts they manage:
+
+* Service account name
+* Type of authenticator used, method of safeguarding and where it is stored
+* Names of all specific individuals with access or knowledge of the service account authenticator
+* Date of last authenticator rotation and reason (ie, 3 years, team member leaving)
+* Intended usage of the service account
+* All applications, services, or processes that rely on or use the service account.
+
+The Platform Security team will collect this documentation from all Account Managers within Platform and maintain it in one location to track all service accounts. 
+
```

Comment:
Is there a way that we can enforce the creation of this document as part of the approval process for new service accounts?

Reply:
Absolutely.  For new service account requests, the process will route to PSEC for approval recommendation and then ultimately to an OCTO fed approver for final approval.  With PSEC in the loop on all requests we would add any new accounts to the tracking mechanism.

Reply:
Incorporated via